plugins/kotlin/j2k/shared/tests/testData/newJ2k/binaryExpression/operatorPrecedence.kt

```diff
@@ -28,7 +28,8 @@ internal object J {
 
         val insideMultiline2 = (1
                 + 1
-                - 0x1234) and (0x1234 ushr 1)
+                - 0x1234
+                ) and (0x1234 ushr 1)
```

Comment:
Is it possible to preserve original formatting?

Reply:
In this case I think the updated formatting is actually closer to the original java which was:
```
int insideMultiline2 = 1
                               + 1
                               - 0x1234
                               & 0x1234 >>> 1;
```